### PR TITLE
Amount page

### PR DESCRIPTION
**Pull request by RetroSabian**

### CODE REVIEW

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+
+    update500 = () => {
+        this.setState({ toAdd:  this.state.toAdd + 500 });
+    };
+
+    render() {
+        return <View style = {Styles.container} >
+            <Text>Current Balance R { this.props.walletUploadState.walletBalanceState.balance }
+            </Text>
+            {/* TODO: Format toAdd text
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: gc

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                    this.setState({
+                        toAdd: text
+                    });
+                }}
+            /> */}
+            <TextInput
+                keyboardType = "numeric"
+                placeholder = {'Type Value here'}
+                onChangeText = { value => {
+                    this.setState({
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: use global state and non-anonymous functions

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                }}
+            />
+            <Text>
+               R {this.state.toAdd}
+            </Text>
+            <View style = {Styles.row}>
+                <Button
+                    title="+R100"
+                    type="clear"
+                    onPress={ () => {
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: indentation

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                        this.update(100);
+                    }
+                    }
+                />
+                <Button style = {Styles.marg}
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: indentation

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                    title="+R200"
+                    type="Clear"
+                    onPress={ () => {
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: indentation

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+                        this.update(200);
+                    }
+                    }
+                />
+                <Button style = {Styles.marg}
+                    title="+R500"
+                    type=""
+                    onPress={ () => {
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: indentation

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+            </View>
+            <Button
+                title = "ADD MONEY"
+                type = "Solid Button"
+                color = "yellow"
+                onPress = { () => {
+                    this.props.loadActions.addMoney(this.state.toAdd);
+                }}              
+            />
+            {this.props.walletUploadState.amountToAdd !== null ?
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/AddMoney.js]: !this.props.walletUploadState.amountToAdd

File: src/Components/MoneyToEWallet/WalletUploadActionTypes.js [hunk 8]
```diff
@@ -0,0 +1,4 @@
+export const LOAD = 'LOAD';
```
Comment (GeromeSchutte) [inline src/Components/MoneyToEWallet/WalletUploadActionTypes.js]: LOAD is too general an action name. The name is not scoped to your feature

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 9]
```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import { View , Text , TextInput, Button } from 'react-native';
+//import TextInputMask from 'react-native-text-input-mask'; part of TODO
```
[the remark this answers is not in the document]
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/AddMoney.js]: removed.

File: src/Components/MoneyToEWallet/AddMoney.js [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as LoadActions from './WalletUploadActions';
+import PropTypes from 'prop-types';
+
+class AddMoney extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
```
[the remark this answers is not in the document]
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/AddMoney.js]: removed

File: src/Components/MoneyToEWallet/AddMoney.js (hunk 1 above)
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/AddMoney.js]: removed

File: src/Components/MoneyToEWallet/AddMoney.js (hunk 2 above)
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/AddMoney.js]: changed

File: src/Components/MoneyToEWallet/AddMoney.js (hunk 3 above)
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/AddMoney.js]: changed

File: src/Components/MoneyToEWallet/AddMoney.js (hunk 7 above)
Comment (RetroSabian) [reply src/Components/MoneyToEWallet/AddMoney.js]: changed

**Pull request merged**